terasoluna-tourreservation-web/src/main/java/org/terasoluna/tourreservation/app/managereservation/ReservationReportPdfStamperView.java

```diff
@@ -123,5 +129,10 @@ protected void prepareResponse(HttpServletRequest request,
         super.prepareResponse(request, response);
         response.setCharacterEncoding(StandardCharsets.UTF_8.name());
     }
-
+    
+    private LocalDate date2LocalDate(final Date date) {
+        return date != null ? Instant.ofEpochMilli(date.getTime())
+                .atZone(ZoneId.systemDefault())
+                .toLocalDate() : null;
```

Comment:
>  Please convert from Date to LocalDateTime.

It was fixed as pointed out  

> `date.toInstant()`

This way is not suitbale here because it was considered as java.sql.Date function and throwed exception.

Reply:
`java.sql.Date` would cause an `UnsupportedOperationException`, so you are correct in your reply. However, I don't see where you are using `java.sql.Date` in this modified section, where is the error?

Reply:
`java.sql.Date` would cause an `UnsupportedOperationException`, so you are correct in your reply. However, I don't see where you are using `java.sql.Date` in this modified section, where is the error?